atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1373,6 +1373,12 @@ void WebContents::SetSize(const SetSizeParams& params) {
     guest_delegate_->SetSize(params);
 }
 
+void WebContents::SetGuestSize(int width, int height) {
+  SetSizeParams params;
+  params.normal_size.reset(new gfx::Size(width, height));
+  SetSize(params);
```

Comment:
This seems like a convenience method on top of the existing `SetSize` method, is there some reason that one can't be used directly for setting the normal size of the webview's web contents?

I'm asking because the docs for `setGuestSize` mention a `webview` tag but this method is on all webContents object and could be called for a `BrowserWindow` web contents, so it might be a bit confusing since this method works even when `IsGuest` would be `false`.